versions.tf

```
@@ -1,9 +1,17 @@
 terraform {
-  required_version = ">= 0.12.0"
-
+  required_version = ">= 0.13.3"
   required_providers {
-    aws      = ">= 2.34"
-    template = ">= 2.0"
-    local    = ">= 1.2"
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.1"
+    }
+    template = {
+      source  = "hashicorp/template"
+      version = ">= 2.0"
+    }
+    local = {
+      source  = "hashicorp/local"
+      version = ">= 1.3"
+    }
```

Preserve Terraform 0.12 compatibility. I also do not see a reason to disallow `local` version 1.2
```suggestion
    aws      = ">= 3.1"
    template = ">= 2.0"
    local    = ">= 1.2"
```

---

I just updated to the latest version, does it have to be 1.2?

---

let's keep 1.2